src/main/java/edu/ucsb/cs156/happiercows/controllers/CommonsController.java

```diff
@@ -89,12 +89,12 @@ public ResponseEntity<String> updateCommons(
     updated.setMilkPrice(params.getMilkPrice());
     updated.setStartingBalance(params.getStartingBalance());
     updated.setStartingDate(params.getStartingDate());
+    //doesn't make sense to update totalPlayer number
```

Comment:
Probably should remove the comment about updating totalPlayer because we should probably remove that as a separate field (replacing with an SQL query).